plot_continuous_monitor.py

```diff
@@ -10,7 +9,7 @@ def write_data(out_file, data):
   out_file.write("\t".join(stringified))
   out_file.write("\n")
 
-def plot_continuous_monitor(filename):
+def plot_continuous_monitor(filename, open_graphs=False):
```

Comment:
Or, would you prefer that the general default is open_graphs=True and only copy_logs sets it to False?